lib/ntbb-session.lib.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-require_once dirname(__FILE__) . '/ntbb-database.lib.php';
+require_once './ntbb-database.lib.php';
+require_once '../config/config.inc.php';
```

Comment:
```suggestion
require_once 'ntbb-database.lib.php';
require_once './config/config.inc.php';
```
I'm not very familiar with PHP paths, but I found that this setup fixed several file not found errors.

Reply:
Seriously???

Reply:
Okay, it turns out I was wrong, my workaround was actually necessary...

```suggestion
require_once __DIR__ . '/ntbb-database.lib.php';
require_once __DIR__ . '/../config/config.inc.php';
```